apps/broomva/app/globals.css

```
@@ -365,6 +389,7 @@
 }
 
 @utility text-balance { text-wrap: balance; }
+@utility text-pretty { text-wrap: pretty; }
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)